.github/workflows/docker.yml

```diff
@@ -57,5 +77,12 @@ jobs:
           push: true
           platforms: linux/amd64, linux/arm64
 
-      - name: List built image(s)
-        run: docker images
```

Comment:
Turns out this step wasn't really doing anything except for listing the images present on GitHub Actions runners